### PR TITLE
Implement `setFreeMonitoring` stub

### LINKED ISSUE #744: Implement `setFreeMonitoring` command (stub)
After #745, implement `setFreeMonitoring` command that would always return an error, just like MongoDB with `--enableFreeMonitoring off` flag. See https://www.mongodb.com/docs/manual/reference/command/setFreeMonitoring/

### PR DESCRIPTION
**Pull request by noisersup**
Closes #744.

# Checklist

* [x] Tests are added for new functionality or fixed bugs.
* [x] `task all` passes.

See CONTRIBUTING.md for more details.

### CODE REVIEW

File: integration/commands_freemonitoring_test.go [hunk 1]
```diff
@@ -53,3 +54,79 @@ func TestCommandsFreeMonitoringGetFreeMonitoringStatus(t *testing.T) {
...
+	ctx, collection := setupWithOpts(t, &setupOpts{
+		databaseName: "admin",
+	})
+
+	for name, tc := range map[string]struct {
+		command  bson.D
+		expected map[string]any
+		err      *mongo.CommandError
+	}{
+		"SetFreeMonitoring_Enable": {
```
Comment (AlekSi) [inline integration/commands_freemonitoring_test.go:69]:
```suggestion
		"Enable": {
```

So the full test name will be `TestCommandsFreeMonitoringSetFreeMonitoring/Enable`

File: integration/commands_freemonitoring_test.go [hunk 2]
```diff
@@ -53,3 +54,79 @@ func TestCommandsFreeMonitoringGetFreeMonitoringStatus(t *testing.T) {
...
+
+func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
+	t.Parallel()
+	ctx, collection := setupWithOpts(t, &setupOpts{
+		databaseName: "admin",
+	})
+
+	for name, tc := range map[string]struct {
+		command  bson.D
+		expected map[string]any
```
Comment (AlekSi) [inline integration/commands_freemonitoring_test.go:66]: It's unused, let's remove it

File: internal/handlers/common/msg_setfreemonitoring.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+)
+
+// MsgSetFreeMonitoring is a common implementation of the setFreeMonitoring command.
+func MsgSetFreeMonitoring(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	action, command := "", document.Command()
```
Comment (AlekSi) [inline internal/handlers/common/msg_setfreemonitoring.go:32]:
```suggestion
	command := document.Command()
	var action string
```

We typically don't assign zero value explicitly. See https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style, for example

But that's a tiny one :)

File: internal/handlers/common/msg_setfreemonitoring.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	switch action {
+	case "enable", "disable":
+		return nil, NewErrorMsg(
+			ErrFreeMonitoringDisabled,
+			"Free Monitoring has been disabled via the command-line and/or config file",
+		)
+	default:
+		return nil, NewErrorMsg(ErrBadValue, fmt.Sprintf(
```
Comment (seeforschauer) [inline internal/handlers/common/msg_setfreemonitoring.go:45]: Please, make the code consistent - `ErrBadValue` should be on the next line as everywhere.

File: internal/handlers/common/msg_setfreemonitoring.go [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+			"Enumeration value '%s' for field '%s' is not a valid value.",
+			action,
+			command+".action",
+		))
+	}
```
Comment (noisersup) [inline internal/handlers/common/msg_setfreemonitoring.go:50]: @seeforschauer something like that would be good?
```suggestion
	switch action {
	case "enable", "disable":
		return nil, NewErrorMsg(
			ErrFreeMonitoringDisabled,
			"Free Monitoring has been disabled via the command-line and/or config file",
		)
	default:
		return nil, NewErrorMsg(
			ErrBadValue,
			fmt.Sprintf("Enumeration value '%s' for field '%s' is not a valid value.",
				action,
				command+".action",
			))
	}

```

Comment (noisersup) [reply internal/handlers/common/msg_setfreemonitoring.go:50]: @AlekSi?
```suggestion
	switch action {
	case "enable", "disable":
		return nil, NewErrorMsg(
			ErrFreeMonitoringDisabled,
			"Free Monitoring has been disabled via the command-line and/or config file",
		)
	default:
		return nil, NewErrorMsg(
			ErrBadValue,
			fmt.Sprintf(
			        "Enumeration value '%s' for field '%s' is not a valid value.",
				action,
				command+".action",
			)
		)
	}

```

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
@rumyantseva @seeforschauer PTAL

**Pull request merged**